addons/stock/models/stock_move.py

```diff
@@ -767,7 +767,7 @@ def _prepare_merge_moves_distinct_fields(self):
         return [
             'product_id', 'price_unit', 'procure_method', 'location_id', 'location_dest_id',
             'product_uom', 'restrict_partner_id', 'scrapped', 'origin_returned_move_id',
-            'package_level_id', 'propagate_cancel', 'description_picking', 'date_deadline'
+            'package_level_id', 'propagate_cancel', 'description_picking',
```

Comment:
We don't want this in the most of the case. It also breaks the new propagation of deadline.

But the client case in the ticket seems correct too (I didn't think about this case before).
@amoyaux proposes (Correct me if I am wrong) that we should instead force the deadline to False in case of `group_propagation_option == 'fixed'` in the `_get_stock_move_values` to manage this case. IMO, it seems a good compromise even if it is a kind of "Monkey-patch".

Reply:
OK, thanks. I still find odd to use a datetime as a grouping field.